test/fixtures/approvals/api_multifile_submission_accepted.approved.json

```diff
@@ -5,6 +5,7 @@
   "slug": "one",
   "name": "One",
   "iteration": 1,
+  "what_next_instructions": "Your fake solution for one has been submitted.\n\nProgrammers generally spend far more time reading code than writing it.\nTo benefit the most from this exercise, find 3 or more submissions that you can\nlearn something from, have questions about, or have suggestions for.\nPost your thoughts and questions in the comments, and start a discussion.\nConsider revising your solution to incorporate what you learn.\n\nYours and others' solutions to this problem:\nhttp://example.org/tracks/fake/exercises/one\n",
```

Comment:
:-)

I'd probably just leave this as the first sentence, to keep the fixture a bit more terse.